.circleci/scripts/sync-static

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+cd "$(dirname $0)/../.."
+
+export AWS_SECRET_ACCESS_KEY
+export AWS_ACCESS_KEY_ID
+
+aws s3 sync \
+    --acl "public-read" \
+    --cache-control "max-age=3600" \
```

Comment:
Is this a standard `max-age` value across our services? We could probably increase this quite a bit? Or does CloudFront set its own?

Reply:
We can set it to whatever you'd like.